packages/itwinui-css/src/surface/surface.scss

```diff
@@ -22,5 +22,63 @@
 
 // Public facing surface component.
 @mixin iui-surface-component {
+  --iui-surface-padding-inline: var(--iui-size-s);
+  --iui-surface-content-padding: var(--iui-surface-padding-inline);
+
   @include iui-surface;
+  display: flex;
+  flex-direction: column;
+  height: var(--iui-surface-content-height);
+  width: 100%; // Copying line 16 of `fluid-grid.scss` in iTwinUI-layouts, unsure if needed
+}
+
+@mixin iui-surface-header-footer($type) {
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  overflow: hidden;
+
+  @if ($type == 'header') {
+    border-bottom: 1px solid var(--iui-surface-border-color);
+    padding: calc(var(--iui-surface-padding-inline) * 0.5) var(--iui-surface-padding-inline);
+  }
+
+  @if ($type == 'footer') {
+    border-top: 1px solid var(--iui-surface-border-color);
+  }
+}
+
+@mixin iui-surface-footer-modifiers {
+  .iui-button {
+    flex-grow: 1;
+    border-radius: 0;
+    @media (forced-colors: active) {
+      border: none;
+    }
+
+    + .iui-button {
+      border-left: 1px solid var(--iui-surface-border-color);
+    }
+
+    &:first-of-type {
+      border-bottom-left-radius: var(--iui-surface-border-radius);
+    }
+
+    &:last-of-type {
+      border-bottom-right-radius: var(--iui-surface-border-radius);
+    }
+  }
+}
+
+@mixin iui-surface-content {
+  display: flex;
+  flex-grow: 1;
+  padding: var(--iui-surface-content-padding);
+  overflow-y: auto;
+  overflow-y: overlay;
+}
+
+@mixin iui-surface-content-empty {
```

Comment:
This might not be needed at all, might be able to use just #1011 instead.

Reply:
then remove it 😅

Reply:
Removed and using a `.demo-flex`.